modules/minimum-required-permissions-ipi-gcp-xpn.adoc

```diff
@@ -25,6 +25,18 @@ Ensure that the host project applies one of the following configurations to the
 * `roles/compute.securityAdmin`
 ====
 
+.Required permissions for creating service account resources
+[%collapsible]
+====
+* `resourcemanager.projects.getIamPolicy`
+* `resourcemanager.projects.setIamPolicy`
+====
+
+[IMPORTANT]
+====
+These permissions are required when the user does not supply a service account for compute and control plane nodes in the install config. The service account(s) should exist in the host project.
```

Comment:
> The service account(s) should exist in the host project.

Let's rephrase this a bit. The service account can be from any project, but the permissions need to be granted for the host project. So it might be more clear to say something like "these permissions should be granted in the host project."